client/app/redux/lib/stateProperties.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * In order to help do auto-complete on the entire
+ * state of our app, save every new reducer 
+ * that you make in here with the variable name
+ * as both the key and the value
+ * so you can use it when you call connectWithRedux()
+ * */
+export default stateProperties = {
+    incrementingValues: 'incrementingValues',
```

Comment:
Again, eventually deadcode

Reply:
I can mark it with a TODO to remove in a few PRs.